# Header
std_msgs/Header header

geometry_msgs/Pose2D cur_pose2d

geometry_msgs/Pose2D target

geometry_msgs/Pose2D[] next_pathpoints

bool is_last_pathpoint
